pilot/proxy/envoy/discovery_test.go

```diff
@@ -226,6 +226,9 @@ func TestClusterDiscoveryCircuitBreaker(t *testing.T) {
 	addConfig(registry, egressRule, t)
 	addConfig(registry, egressRuleCBPolicy, t)
 
+	// add an egress rule for TCP to an external service (cloud.google.com, by IP range)
+	addConfig(registry, egressRuleTCP, t)
+
```

Comment:
you can remove these. You don't need to add this to every case. Just add to a few. Specifically for the ones with auth

Reply:
please remove this